queries/attachments/auto_executing_macros.yml

```diff
@@ -0,0 +1,15 @@
+name: "Attachment with auto-executing VBA macros"
```

Comment:
Nit: these should all be plural, so "Attachments with auto-executing VBA macros"